docker/exitcode-test.yaml

```diff
@@ -0,0 +1,24 @@
+version: '3.6'
```

Comment:
can we just run all the tests together at the same time and not have a separate docker-compose ?

we probably need to implement a testing strategy first as a separate MR ;-)